src/index.js

```diff
@@ -17,15 +17,14 @@ const {
 } = require('./services')
 const actions = require('./actions')
 const middlewares = require('./middlewares')
+const { applyRoutes } = require('./routes')
 
 const app = express()
 const server = http.createServer(app)
 const io = socketio(server)
 
 app.use(middlewares)
-
-app.get('/rooms', actions.getRooms)
-app.get('/room/:roomId/messages', actions.getRoomMessages)
+applyRoutes(app, actions)
```

Comment:
Is it some sort of a reserved word for app to use? Or does it just use whatever is exported in that file?

Reply:
actions? this is kinda part of MVC pattern